pkg/printutil/printutil.go

```diff
@@ -136,3 +170,40 @@ func strSliceToInterSlice(ss []string) []interface{} {
 	}
 	return is
 }
+
+func (t *Table) createHeaderPadding() {
+	if len(t.headerPadding) == 0 {
+		for _, num := range t.Header {
+			colLength := len(num) + 5
+			t.headerPadding = append(t.headerPadding, colLength)
+		}
+	}
+}
+
+func (t *Table) coalesceHeaderAndRowPadding(val int, i int) {
+	if val < t.headerPadding[i] {
+		t.altPadding[i] = t.headerPadding[i]
+	} else {
+		t.headerPadding[i] = t.altPadding[i]
+	}
+}
+
+func (t *Table) dynamicPadding(rows []TempRow) {
+	for _, row := range rows {
+		for i, col := range row.Values {
+			colLength := len(col) + 5
+			if len(t.altPadding) != len(row.Values) {
+				t.altPadding = append(t.altPadding, colLength)
+			} else {
+				if t.altPadding[i] < colLength {
```

Comment:
What is case covering?

Reply:
If the altPadding slice is smaller than the number of incoming Values slice, then that means that values still need to be added to fill it up. 

If the altPadding slice length is equivalent to the Values slice length, then it should compare the length of the incoming value being evaluated to see if it longer than the value already stored in it's place in altPadding. If it is, replace it's value with the length of the incoming value.

This helps ensure as it iterates through each row that the length of the longest value for each column is kept or replaced as new values are introduced.

Reply:
I added ^^ as a comment to the code itself.